nucypher/blockchain/eth/actors.py

```diff
@@ -390,45 +382,46 @@ def _resolve_validators(
         result = sorted(result, key=lambda x: x[0].address)
         return result
 
-    def publish_transcript(self, ritual_id: int, transcript: Transcript) -> TxReceipt:
+    def publish_transcript(self, ritual_id: int, transcript: Transcript) -> HexBytes:
         """Publish a transcript to publicly available storage."""
         # look up the node index for this node on the blockchain
-        receipt = self.coordinator_agent.post_transcript(
+        tx_hash = self.coordinator_agent.post_transcript(
             ritual_id=ritual_id,
             transcript=transcript,
-            transacting_power=self.transacting_power
+            transacting_power=self.transacting_power,
+            fire_and_forget=True,
         )
-        return receipt
+        return tx_hash
 
     def publish_aggregated_transcript(
         self,
         ritual_id: int,
         aggregated_transcript: AggregatedTranscript,
-        public_key: FerveoPublicKey,
-    ) -> TxReceipt:
+        public_key: DkgPublicKey,
+    ) -> HexBytes:
         """Publish an aggregated transcript to publicly available storage."""
         # look up the node index for this node on the blockchain
         participant_public_key = self.threshold_request_power.get_pubkey_from_ritual_id(
             ritual_id
         )
-        receipt = self.coordinator_agent.post_aggregation(
+        tx_hash = self.coordinator_agent.post_aggregation(
             ritual_id=ritual_id,
             aggregated_transcript=aggregated_transcript,
             public_key=public_key,
             participant_public_key=participant_public_key,
-            transacting_power=self.transacting_power
+            transacting_power=self.transacting_power,
+            fire_and_forget=True,
```

Comment:
I initially didn't get what `fire_and_forget` does, I call this mode "block until complete",

Reply:
I guess you mean block until complete is when fire and forget is false, right? Because fire and forget = true is the opposite of blocking